nav2_bringup/bringup/launch/nav2_bringup_launch.py

```diff
@@ -29,15 +30,32 @@ def generate_launch_description():
     launch_dir = os.path.join(bringup_dir, 'launch')
 
     # Create the launch configuration variables
+    namespace = LaunchConfiguration('namespace')
     map_yaml_file = LaunchConfiguration('map')
     use_sim_time = LaunchConfiguration('use_sim_time')
     params_file = LaunchConfiguration('params_file')
     bt_xml_file = LaunchConfiguration('bt_xml_file')
     autostart = LaunchConfiguration('autostart')
+    use_remappings = LaunchConfiguration('use_remappings')
+
+    # TODO(orduno) Remove once `PushNodeRemapping` is resolved
+    #              https://github.com/ros2/launch_ros/issues/56
+    remappings = [((namespace, '/tf'), '/tf'),
+                  ((namespace, '/tf_static'), '/tf_static'),
+                  ('/scan', 'scan'),
```

Comment:
`/scan` is currently being set via the yaml file, could we also just set the parameter name as `scan` in the `nav2_params.yaml`?

Reply:
Once this gets merged I am planning on setting scan, map, etc. on a relative namespace.

Reply:
And remove the corresponding remappings from the launch file.